app/views/topics/show.html.erb

```diff
@@ -1,5 +1,7 @@
 <h1><%= @topic.name %></h1>
- <% if user_is_authorized_for_topics? %>
+ <% if user_is_authorized_for_edit? %>
+   <%= link_to "Edit Topic", edit_topic_path, class: 'btn btn-success' %>
+ <% elsif user_is_authorized_for_topics? %>
    <%= link_to "Edit Topic", edit_topic_path, class: 'btn btn-success' %>
```

Comment:
If you break this up into two `if` statements (instead of an `else`) you won't need to repeat this line